### PR TITLE
[Bugfix]Aligning select color tag in 'Edit categories'

### LINKED ISSUE #3461: Misalignment on editing categories
## Issue
Misalignment of color select tag when name of categories is selected in Firefox browser works fine in google chrome
ezgif com-video-to-gif(1)

### PR DESCRIPTION
**Pull request by drumilpatel2000**
Fixes #3461
ezgif com-video-to-gif(2)

### CODE REVIEW

File: site/public/css/server.css [hunk 1]
```diff
@@ -1270,6 +1270,7 @@ div.full_height {
     border-radius: 0.3em;
     margin: 0.3em 0.1em 0.3em 0.1em;
     padding: 0.2em;
+    width:400px;
```
Comment (andrewaikens87) [inline site/public/css/server.css]: Can we simply just modify the width on line 1263? Also I didn't experience this issue, are you zoomed in?

Comment (drumilpatel2000) [reply site/public/css/server.css]: This issue is only visible on firefox browser

**Comment by drumilpatel2000** [comment]
@andrewaikens87  Can you please review it again?

**Comment by drumilpatel2000** [comment]
@andrewaikens87 Is this okay?

**Pull request merged**